For now keep using base ld.bfd even if installing binutils...

--- Makefile.orig	2019-09-04 12:05:52 UTC
+++ Makefile
@@ -13,6 +13,7 @@ MASTER_SITES=	ftp://ftp.freepascal.org/p
 		ftp://ftp.us.freepascal.org/pub/fpc/dist/${PORTVERSION}/source/:source \
 		SF/freepascal/Source/${PORTVERSION}:source \
 		http://leaf.dragonflybsd.org/~marino/dports-src/:DragonFlybootstrap \
+		http://avalon.dragonflybsd.org/misc/distfiles/freepascal/:DragonFlybootstrap \
 		LOCAL/acm/freepascal/:bootstrap \
 		LOCAL/acm/freepascal/:man
 DISTFILES=	${DISTNAME:S/$/.source/}${EXTRACT_SUFX}:source
@@ -28,7 +29,7 @@ USE_BINUTILS=	yes
 BOOTVER=	3.0.4
 MANVER=		3.0.4
 
-LDPATH=		"${LOCALBASE}/bin/ld.bfd"
+LDPATH=		"/usr/bin/ld.bfd"
 
 #USE_GITHUB=	yes
 #GH_ACCOUNT=	fpc-svn
@@ -126,7 +127,7 @@ do-build:
 # build fpc compiler
 	@${ECHO_MSG} "##### STARTING COMPILER #####"
 	(cd ${WRKDIR}/${FPCSRCDIR}/compiler && ${SETENV} ${MAKE_ENV} \
-		${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
+		LDVER=ld.bfd ${MAKE_CMD} cycle ${MAKE_ARGS} ${BOOTPPC})
 	@${ECHO_MSG} "##### COMPLETE COMPILER #####"
 
 # build fpcmkcfg
